Gemfile.lock

```diff
@@ -281,4 +282,4 @@ DEPENDENCIES
   whenever
 
 BUNDLED WITH
-   2.2.20
+   2.2.33
```

Comment:
It's up to you if you want to change this and it may not be an issue, but to be safe we might want to keep the bundler at 2.2.20.  If it deploys and there are no issues, then I think it's fine.

Reply:
Oh weird, I missed that. It must have happened when I ran bundle install to get tract up to date. I'll change it back.